building/trusted_applications.rst

```diff
@@ -373,35 +373,50 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the in-tree TAs, which come
+with the OP-TEE OS (for example the ``pkcs11`` TA) and any external TA coming from the user. Offline signing
+runs directly on `.stripped.elf` TA files and there is a slight difference as how they are
```

Comment:
runs directly -> operates

Reply:
I think, the second half of the sentence ("..and there is .. generated") can be dropped.

Reply:
You are right, I changed the first paragraph a little and I think it's more clear now.